zulipterminal/ui_tools/views.py

```diff
@@ -1939,14 +1939,21 @@ def __init__(
 
         # Get rendered message header and footer
         msg_box = MessageBox(message, controller.model, None)
+        msg_box.footer = [
+            (
+                urwid.Text(
+                    ("msg_bold", "\nPress [c] to copy message to clipboard"),
```

Comment:
For specific keys, we always want to avoid hard-coding the key in the text, and rather lookup the key name via it's symbolic name: `COPY_MESSAGE`.

Reply:
Yes that makes more sense! Making that change.